docs/sources/auth/enhanced_ldap.md

```diff
@@ -41,3 +41,11 @@ a user as member of a team and it will not be removed when the user signs in. Th
 5. Click on `Add group` button to save.
 
 <div class="clearfix"></div>
+
+## Active LDAP Synchronization
+
+With the open source version of grafana LDAP users will be sync only during the log in process.
+
+With this feature you can configure grafana to actively sync grafana users with LDAP server(s) in the background. After certain configurable period it will map the groups with the already existing grafana users, disable removed LDAP users and etc.
```

Comment:
s/grafana/Grafana
s/sync/synchronize

Etc what?

Reply:
> s/sync/synchronize

Why we can't use abbreviation? 

> Etc what?

I was intentionally making it vague, since there might be new stuff added or removed, so instead of providing comprehensive item list I thought it would be a good idea to keep out the specifics

Reply:
> Why we can't use abbreviation?

Personal taste, but think it's get easier to read and understand when being explicit in documentation.

> I was intentionally making it vague, since there might be new stuff added or removed, so instead of providing comprehensive item list I thought it would be a good idea to keep out the specifics

Whenever we add/remove new stuff we should update the documentation instead. I think it's much better to have a clear documentation that may lack information than have a vague documentation. I think we also have a good opportunity to explain here why someone want to use this this feature. But I guess we can make this better in the future.

Maybe it's the "and etc" that confuses me so at least skip the "and".

Reply:
Well, for me those are implementation details, I wanted to give couple examples so it would be more clear to see what is meant by "user sync" process